<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ethernet_connection_widget_form</class>
 <widget class="QWidget" name="Ethernet_connection_widget_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="address_edit_">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="port_spinbox_">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="connect_button_">
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>105</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
